### ISSUE TITLE
Flatpak package

### ISSUE THREAD

**Issue by 13hannes11** [hakandundar34coding/system-monitoring-center#15]
It would be great to have a flatpak packae for this tool.

**Comment by hakandundar34coding** [comment]
I prepared ```tr.org.pardus.system-monitoring-center.yml``` file for preparing Flatpak package for System Monitoring Center. This application is a detailed system application and it was not easy to prepare Flatpak package. I do not know some details of preparing Flatpak package.

A package can be generated by using ```tr.org.pardus.system-monitoring-center.yml``` file but some dependencies are missing. The application can be started but some tabs do not work.

A Flatpak package may be prepared without these problems if contributors prepare this package.

**Comment by hakandundar34coding** [comment]
I have updated the ```yml``` file and its name is: ```com.github.hakand34.system-monitoring-center.yml```.
This file can be found on the main directory of the project.

There are improvements for Flatpak packaging but there are also problems. Some of the problems may not be fixed. For example, listing and managing processes. As far as i know, Flatpak applications can not access to processes of the OS. These applications can access to processes in the sandbox. This is limitatation of the Flatpak.

I did not see any task managers or system monitors on Flathub. The reason for this situation may be the same.

Currently, I have no plans for fixing all problems of Flatpak package of the System Monitoring Center.
